### DISCUSSION TITLE
Modifying `reading_correction.tsv`

### DISCUSSION THREAD

**Discussion by tats-u** [google/mozc#1058] [Q&A] [answered]
May we create PRs to modify `src\data\dictionary_oss\reading_correction.tsv`?
I'll add:

```
上側	うえがわ	うわがわ
鶴舞公園	つるまいこうえん	つるまこうえん
```

**Comment by hiroyuki-komatsu** [comment, marked as answer]
Thank you for your PR for this discussion!
https://github.com/google/mozc/pull/1069
